server/src/main/java/org/apache/iotdb/db/engine/compaction/cross/CrossSpaceCompactionTask.java

```diff
@@ -149,7 +149,7 @@ protected void doCompaction() throws Exception {
       }
     } catch (Throwable throwable) {
       // catch throwable instead of exception to handle OOM errors
-      LOGGER.error("Meet errors in cross space compaction, {}", throwable.getMessage());
+      LOGGER.error("Meet errors in cross space compaction.");
```

Comment:
log the exception?

Reply:
The exception is caught by `AbstractCompactionTask.call()` and logged in error level. So there's no need to log twice.